### DISCUSSION TITLE
What does "was" on Commit page mean?

### DISCUSSION THREAD

**Discussion by dimiterbak** [community/community#21618] [New to GitHub]
<p>When I open a Commit page e.g. <a href="https://github.com/wildfly/wildfly-core/commit/434b31592ce4ea9729943b3bfbe0894d516707c0" class="inline-onebox" rel="noopener nofollow ugc">Interface support · wildfly/wildfly-core@434b315 · GitHub</a></p>
<p>there is under the commit message a label:  “was: 9445bfa”</p>
<p>What does that “was:” mean?</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>That is part of the commit message, not under the commit message. So if you need to know what it means it’s probably best to ask the author of that commit.</p>
